internal/statestore/ticket_test.go

```diff
@@ -901,6 +901,7 @@ func createRedis(t *testing.T, withSentinel bool, withPassword string) (config.V
 	cfg.Set("redis.pool.idleTimeout", time.Second)
 	cfg.Set("redis.pool.healthCheckTimeout", 100*time.Millisecond)
 	cfg.Set("redis.pool.maxActive", 5)
+	cfg.Set("redis.mutexTimeout", "5m")
```

Comment:
Once the test passes, will try deleting this change, because i'm not sure if it is useful here

Reply:
Backfill also uses this, so leaving it as it is